setup.py

```diff
@@ -37,6 +37,16 @@
     packages=['dukpy', 'dukpy.webassets'],
     ext_modules=[duktape],
     install_requires=INSTALL_REQUIRES,
+    extras_require={
+        'testing': [
```

Comment:
This would allow users to install test deps along with project if they want to run tests themselves instead of manually copy-pasting commands from Travis config. It's easier to maintain a single place of requirements.